### PR TITLE
added index.html

### PR DESCRIPTION
**Pull request by BayliSade**

### CODE REVIEW

**Comment by BayliSade** [comment]
added index.html

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (shahrzad88) [inline index.html]: Change the title.

Comment (jojo07206) [inline index.html]: hi

File: index.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+</head>
+<body>
+
```
Comment (shahrzad88) [inline index.html]: Add the header.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (jojo07206) [inline index.html]: hi

File: index.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+<html lang="en">
```
[the remark this answers is not in the document]
Comment (shahrzad88) [reply index.html]: gfgcnh

**Pull request merged**